.pylintrc

```diff
@@ -212,7 +212,21 @@ good-names=FlightPhases,
            fin_set_NACA,
            fin_set_E473,
            HIRESW_dictionary,
-
+           prop_I_11,
+           Kt, # transformation matrix transposed
+           clalpha2D, 
+           clalpha2D_incompresible,
+           r_NOZ, # Nozzle position vector
+           rocket_dry_I_33,
+           rocket_dry_I_11,
+           motor_I_33_at_t,
+           motor_I_11_at_t,
+           motor_I_33_derivative_at_t,
+           motor_I_11_derivative_at_t,
+           M3_forcing,
+           M3_damping,
+           CM_to_CDM,
+           CM_to_CPM,
```

Comment:
What you suggest sounds like "let's ignore any variables that has one or more uppercase letters".

But the point is: **you should not add uppercase variables in the code**.

If you do, pylint will warn you about it. I added here the "exceptions" because I can't simply rename all the current variables that are not following the `snake_case` convention. I understand that sometimes it may be good idea to use uppercase letters, but then we can discuss - case by case - if we are going to add more exceptions or not.

Ideally, the list of exception variables should decrease over time, not increase.

This is annoying, of course. I think it is important for the class attributes. Final users don't really access the local variables. But I wouldn't risk adding non snake-case to the public methods and arguments.

Reply:
I even avoided to use regex, not only because **I believe** it is slower, but also because I wanted to explicitly points what are the "exceptions for the snake_case rule"